sdk-python/prisma/schema.prisma

```diff
@@ -0,0 +1,19 @@
+datasource client {
+  provider = "sqlite"
```

Comment:
We're currently using postgres to test for now (included in Definition of done I believe)